justfile

```diff
@@ -113,15 +112,16 @@ fix: devenv
 
 
 # Run the dev project
-run:
+run: 
```

Comment:
did you mean to add a trailing space here?

Reply:
bah